backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -117,7 +119,7 @@ private Specification<TestEvent> buildTestEventSearchFilter(
                     root.get(BaseTestInfo_.patient).get(AuditedEntity_.internalId), patientId));
       }
       if (result != null) {
-        p = cb.and(p, cb.equal(root.get(BaseTestInfo_.result), result));
+        p = cb.and(p, cb.equal(resultJoin.get(Result_.testResult), result));
```

Comment:
@kenieh a question for you here. This backend change will affect the frontend filters on the Result page such that filtering by result (positive, negative, inconclusive) will now show ALL test results that match, instead of showing only the covid results as currently happens. I believe that's what your designs called for, but can you confirm?